clinical/example-params.csv

```diff
@@ -0,0 +1,7 @@
+qrcode,pat_name,birth_date,collect_ts,result_ts,status_code,swab_type,staff_observed,pre_analytical_specimen_collection,ordering_provider
+AAAAAAAA,Not-received Ned,2020-01-01,,,not-received,mtb,t,IRB,"Example Provider MD MPH"

```

Comment:
Technically, we shouldn't have an ordering_provider unless the pre_analytical_specimen_collection is 'clinical', so remove the the ordering_provider value for the rows that are IRB.

Reply:
That's fair. How about I change all these to clinical since that's the template it's intended to go with? I want to generate mock PDFs for the different types of results / status_codes

Reply:
That works. I added the two new fields to the scan-irb example-parms.csv file too. I made that one include both IRB and clinical, so please remove the "clinical" ones from there to be consistent--to have the example CSV be either all clinical or all IRB.

Reply:
Updated in latest push.